chan_handler.go

```diff
@@ -1,10 +1,12 @@
-// Copyright 2013-2016 Apcera Inc. All rights reserved.
+// Copyright 2013-2017 Apcera Inc. All rights reserved.
 
 package graft
 
 // ChanHandler is a convenience handler when a user wants to simply use
 // channels for the async handling of errors and state changes.
 type ChanHandler struct {
+	LogPositionHandler
```

Comment:
I wish I could think of something better, but can't.  There isn't full RAFT spec log replication, but the value passed is actually the position in the log in the context of leadership election.  If not a name change, I think a comment to that effect might help clarify.